src/App.js

```diff
@@ -22,8 +23,7 @@ const App = ({ fetching }) => {
 
   const getFeed = (event) => {
     setFetching((prev) => !prev);
-    if (event.preventDefault != null)
-      event.preventDefault();
+    if (event.preventDefault != null) event.preventDefault();
```

Comment:
We can change the condition to be `if (event.preventDefault)`

Reply:
It would be clear to compare it with `null` instead of implicitly assume that !